app/models/miq_report/formats.rb

```diff
@@ -1,19 +1,27 @@
 class MiqReport::Formats
-  format_hash = YAML.load_file(ApplicationRecord::FIXTURE_DIR.join('miq_report_formats.yml')).freeze
```

Comment:
` Using user input when accessing files (local or remote) will raise a warning in Brakeman.`

How is this even a warning with brakeman?   constant + string should be pretty obvious it's not user input?  I guess they're not checking what the constant's value is and if it come from user input at runtime?

So weird.

Reply:
I really don't understand it myself.